src/components/LiquidityMining/Create/Steps/RewardAmount/index.tsx

```diff
@@ -1,175 +1,148 @@
-import { Pair, TokenAmount } from '@swapr/sdk'
-import React, { useCallback, useEffect, useState } from 'react'
-import { Box, Flex } from 'rebass'
-import styled from 'styled-components'
-import { TYPE } from '../../../../../theme'
-import { CurrencyLogo } from '../../../../CurrencyLogo'
-import NumericalInput from '../../../../Input/NumericalInput'
-import Radio from '../../../../Radio'
-import { Card, Divider } from '../../../styleds'
-import border8pxRadius from '../../../../../assets/images/border-8px-radius.png'
-import DoubleCurrencyLogo from '../../../../DoubleLogo'
-import { tryParseAmount } from '../../../../../state/swap/hooks'
-import { usePrevious } from 'react-use'
-
-const RelativeContainer = styled.div<{ disabled?: boolean }>`
-  position: relative;
-  transition: opacity 0.3s ease;
-  opacity: ${props => (props.disabled ? 0.5 : 1)};
-`
-
-const StyledNumericalInput = styled(NumericalInput)`
-  border: 8px solid;
-  border-radius: 8px;
-  border-image: url(${border8pxRadius}) 8;
-  width: 100%;
-  height: 33px;
-  font-weight: 600;
-  font-size: 16px;
-  line-height: 16px;
-  text-transform: uppercase;
-  padding-left: 8px;
-  padding-right: 8px;
-  background-color: ${props => props.theme.dark1};
-`
-
-const RewardInputLogo = styled(CurrencyLogo)`
-  position: absolute;
-  top: 8px;
-  right: 16px;
-`
+import { Currency, currencyEquals, TokenAmount } from '@swapr/sdk'
+import React, { useCallback, useMemo, useState } from 'react'
+import { Flex } from 'rebass'
 
-const StakablePairInputLogoContainer = styled.div`
-  position: absolute;
-  top: 8px;
-  right: 16px;
-`
+import { tryParseAmount } from '../../../../../state/swap/hooks'
 
-const FlexContainer = styled(Flex)`
-  ${props => props.theme.mediaWidth.upToExtraSmall`
-    flex-direction: column;
-  `}
-`
+import AssetSelector from '../PairAndReward/AssetSelector'
+import {
+  Actions,
+  ActionType,
+  CampaignType,
+  numberOfRewards,
+  RewardsObject,
+} from '../../../../../pages/LiquidityMining/Create'
+import { CurrencySearchModal } from '../../../../SearchModal/CurrencySearchModal'
+import styled from 'styled-components'
+import { ApprovalState } from '../../../../../hooks/useApproveCallback'
 
-const PoolSizeContainer = styled(Box)`
-  ${props => props.theme.mediaWidth.upToExtraSmall`
-    margin-top: 16px !important;
+const FlexWrapper = styled(Flex)`
+  gap: 28px;
+  margin-top: 32px !important;
```

Comment:
No `!important` please. If you need to override styles import from `rebass/styled-components` instead